common/flatpak-dir.c

```diff
@@ -11498,6 +11498,7 @@ flatpak_dir_update_remote_configuration_for_state (FlatpakDir         *self,
     "xa.gpg-keys",
     "xa.redirect-url",
     "xa.collection-id",
+    "eos.collection-id", /* Endless specific and temporary */
```

Comment:
Support for `eos.collection-id` also needs to be added in `app/flatpak-builtins-repo.c`. You might also need to add support for it in `flatpak_repo_update()` in `common/flatpak-utils.c`. Certainly the comment there (which mentions `xa.collection-id`) should be updated.